client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-complete/index.tsx

```diff
@@ -41,6 +42,10 @@ const Complete: Step = function Complete( { flow } ) {
 		dispatch( fetchUserPurchases( userId ) );
 	}, [] );
 
+	const clearDomainsStore = () => {
+		setDomainsTransferData( {} );
```

Comment:
Maybe just call `resetOnboardStore`?

Reply:
This is perfect, thank you for pointing it out. Using it.